src/main/java/ezschedule/commons/core/index/Index.java

```diff
@@ -22,6 +22,9 @@ private Index(int zeroBasedIndex) {
 
         this.zeroBasedIndex = zeroBasedIndex;
     }
+    public static Index initIndex(int zeroBasedIndex) {
+        return new Index(zeroBasedIndex);
+    }
```

Comment:
What is the purpose of this? I can't seem to find where in this PR is this being used.